setup.cfg

```diff
@@ -1,8 +1,5 @@
 [aliases]
 release = egg_info -RDb ''
-# Make sure the sphinx docs are built each time we do a dist.
```

Comment:
Not sure how to find an alternative to this

Reply:
We don't need this now that we use rtd.